install/overlays/libvirt/kustomization.yaml

```diff
@@ -7,7 +7,7 @@ bases:
 images:
 - name: cloud-api-adaptor
   newName: quay.io/confidential-containers/cloud-api-adaptor # change image if needed
-  newTag: latest
+  newTag: dev-d4496d008b65c979a4d24767979a77ed1ba21e76
```

Comment:
nit: do we need the `dev-` prefix for libvirt?

Reply:
Yes - the dev image using different cgo bindings I think, so only works with the dev images I think: https://github.com/confidential-containers/cloud-api-adaptor/blob/d4496d008b65c979a4d24767979a77ed1ba21e76/.github/workflows/caa_build_and_push.yaml#L68-L72

Reply:
yes @mkulke. `latest` and `dev` are built with libvirt-dev. The libvirt go binary cannot be statically linked with the libvirt-dev, so it should distributed and we wanted to avoid shipping the lib on the environment for the other providers.

Reply:
yup I remember stumbling over this :)